### PR TITLE
Update `People/Users_with_unique_titles` with osu!trivium Quiz contest winners

### PR DESCRIPTION
**Pull request by Walavouchey**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

[results livestream vod](https://www.twitch.tv/videos/1237832029?t=02h43m18s)

### CODE REVIEW

File: wiki/People/Users_with_unique_titles/en.md [hunk 1]
```diff
@@ -189,6 +189,15 @@ The osu!remix contests award the **osu!mixer** title as a first-place prize.
 
 ¹ ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) was awarded with the **osu!mixer II** title instead for having won both the [osu!remix contest #1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) and the [A Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) contest.
 
+## Current osu!trivium Quiz contest winners
+
+The osu!trivium Quiz contests award its winning team or individual the **osu!trivium Champion** title, which lasts for one year. These are the title holders for 2021:
```
Comment (TicClick) [inline wiki/People/Users_with_unique_titles/en.md:194]: I'd plug a link to a relevant newspost somewhere here

**Review by TicClick — COMMENTED** [review]
on top of my suggestion, why not use `outdated_since: ...` instead, as it's more precise?

File: wiki/People/Users_with_unique_titles/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/People/Users_with_unique_titles/en.md:194]: would be more appropriate in its own contest article

Comment (Walavouchey) [reply wiki/People/Users_with_unique_titles/en.md:194]: there was also no results news post, just a livestream (and no official tweet even)

Comment (TicClick) [reply wiki/People/Users_with_unique_titles/en.md:194]: well, having a link even to an opening newspost would be good imo

**Review by TicClick — APPROVED** [review]

**Pull request merged**